examples/network_functions/go-gtp/enb/enb_default.yml

```diff
@@ -15,6 +15,6 @@ subscribers:
     src_ip: "10.0.0.201"
     i_tei: 0
     type: "external"
-    euu_if_name: "eth1"
+    euu_if_name: "eth0"
```

Comment:
I see this potentially causing issues if eth0 is the default interface assigned to a pod.
I think most example use_cases generate the files through a configmap, in which case it shouldn't be noticeable, but is there otherwise a reason to update this value?

Reply:
@michaelspedersen I like the idea of using a configmap.  Could that be a followup PR?

@electrocucaracha, @michaelspedersen  does the ENB create an alias interface on the interface specified (eg. eth0 -> eth0:1) or does it "take over" that interface with the IP assignment?  If the former then using eth0 allows this to run on an node with only 1 interface.  If the later I agree it could problematic with the default.

Reply:
I think it takes over the interface. All use-case examples I've added in there at least has 1 interface per IP assignment ([example](https://github.com/cncf/cnf-testbed/blob/master/examples/use_case/gogtp-k8s/k8s_bridged_networks/gogtp/values.yaml))

Reply:
> @michaelspedersen I like the idea of using a configmap. Could that be a followup PR?

Most (all?) use cases already do, but I haven't sifted through all examples to see if any still use the files added when the image is built

Reply:
This change depends on the [order of the networks](https://github.com/electrocucaracha/cnf-testbed/blob/gtp_e2e/examples/use_case/gogtp-docker/docker-compose.yml#L20-L26) defined in the `docker-compose.yml` file. I'm going to modify that to avoid this change.

Reply:
I tested this changing the order in the docker file and also using `priority` [feature](https://github.com/docker/compose/issues/4645#issuecomment-386406667) but the NICs are still assigned in a different order. I can duplicate those configuration files to avoid conflicts.